### PR TITLE
Added in getters and setters for Mapper parameters

### PR DESCRIPTION
**Pull request by lucbettaieb**

### CODE REVIEW

File: src/Mapper.cpp [hunk 1]
```diff
@@ -1799,7 +1799,249 @@ namespace karto
...
+  bool Mapper::getParamUseScanBarycenter(){
+    return (bool)m_pUseScanBarycenter->GetValue();
+  }
+
+  double Mapper::getParamMinimumTravelDistance(){
+    return (double)m_pMinimumTravelDistance->GetValue();
+  }
+
+  double Mapper::getParamMinimumTravelHeading(){
+    return math::RadiansToDegrees((double)m_pMinimumTravelHeading->GetValue());
```
Comment (mikeferguson) [inline src/Mapper.cpp:1838]: ROS standardizes on radians, see http://www.ros.org/reps/rep-0103.html, so even though Karto uses degrees for their default value, I would suggest we stick with radians as the input/output of these functions

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,12 @@
 Changelog for package open_karto
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.1.2 (2015-06-02)
+------------------
+* Added in getters and setters for the Mapper object
```
Comment (mikeferguson) [inline CHANGELOG.rst]: this is not needed -- is autogenerated

**Comment by lucbettaieb** [comment]
Made the appropriate changes through my latest commit.
https://github.com/lucbettaieb/open_karto/commit/aba481c49c1a2775f5ed2ebb2f278f195b257626

**Comment by lucbettaieb** [comment]
Closed to re-do properly with a branch and rebasing.

**Pull request closed without merging**